### ISSUE TITLE
Missing information in problem_list.csv

### ISSUE THREAD

**Issue by shubhamrgandhi** [IBM/Project_CodeNet#57]
Hello, 

I can't seem to be able to find information on rating, tags and complexity in the problem_list.csv file uploaded with the metadata. Would it be possible to obtain an updated copy (or steps on how to obtain information especially on tags). This information would be helpful for my work. 
[problem_list.csv](https://github.com/IBM/Project_CodeNet/files/12117399/problem_list.csv)

**Comment by geert56** [comment]
Isn't that information available in the per problem metadata?

**Comment by shubhamrgandhi** [comment]
Not really, no, the per problem metadata looks something like this and does not have a `tags` field.
[p00002.csv](https://github.com/IBM/Project_CodeNet/files/12141920/p00002.csv)
